src/RegisterMapping.h

```diff
@@ -132,13 +138,16 @@ class RegisterMapping
      * @param readRestricted indicates whether or not the mapping can be read
      * @param slaveAddress of the devices being accessed, leave as default on -1
      * @param deadbandValue indicates a change in value of the register that is insignificant data
+     * @param frequencyFilterValue changes that occur within the given time (in miliseconds) that will be ignored
      */
     RegisterMapping(const std::string& reference, RegisterType registerType, const std::vector<int32_t>& addresses,
                     OutputType type, OperationType operation, bool readRestricted = false, int16_t slaveAddress = -1,
-                    double deadbandValue = 0.0);
+                    double deadbandValue = 0.0, unsigned long long frequencyFilterValue = 0);
 
     virtual ~RegisterMapping() = default;
 
+    static unsigned long long int currentRtc();
```

Comment:
Once you replace types of filter and lastUpdateTime there is no need for this method
Just call std::chrono::time_point\<std::chrono::high_resolution_clock\>::now() where needed